### DISCUSSION TITLE
Planning visualisation tool - Questions for GLA’s Planning DataHub

### DISCUSSION THREAD

**Discussion by fallipalaiologou** [tomalrussell/colouring-london#930] [Q&A]
### Questions for discussion with GLA on CLPV Tool (project described in the colouring cities manual [here](https://github.com/colouring-cities/manual/wiki/G.-Data-capture-methods#live-streaming-and-visualisation-planning-application-data); for tool design brief see #884).

**Questions on data & streaming methods:**

1. In our review of available datasets via the DataHub, we have noted partial data for a small number of councils and only two or three councils appear to be updating their data on the DataHub. How many councils are actively sending live data? How reliable is the streaming service at the moment? 
2. Is the process of uploading/updating data managed manually by the council or is it automated? How often do councils update their data? 
3. It is unclear what percentage of planning applications are being streamed and the extent to which minor applications are being omitted? Our plan is to design a system that visualises all planning applications.
4. Do data updates contain info only on new applications/decisions or do they contain data on all active applications? [this relates to our issue whereby latest data uploads seem to include only appeals and rejections]
5. Does the dataset capture live applications under "submitted status" and for which a decision has not yet been taken? And is there a “field” that records data on application submissions, e.g. a submission date? If not, is the "validation date" an adequate proxy? [Still, validation date only appears for entries with a recorded decision of either Approved/Appeal In Progress/Rejected/Withdrawn/Unknown/Other.]
7. What are planning codes “No Objection to Proposal (OBS only)” and “SECS”? Are planning codes the same for all boroughs? 
8. Is polygon data already available? (perhaps we missed a way to obtain it)
9. If not, is it possible to make polygon data available? Note that in data that @matkoniecz managed to obtain from the DataHub there is an entry for "polygon": null, which seems to indicate that publishing polygon data was supposed to happen. Attempts to obtain data, including polygon data are documented at https://github.com/colouring-cities/planning-datahub-investigation

**Questions on CLPV tool design:**

1. Is the “Disclaimer” information appropriate - any edits required?
2. Does the current visualisation appear to reflect accurately the number of boroughs monitored via the DataHub?
3. Is there any other critical information that is available and we have missed it?
4. What might be the potential impact/value of us adding: a) the [INSPIRE land parcel dataset](https://use-land-property-data.service.gov.uk/datasets/inspire/download); and b) the borough boundaries and names?

**Comment by matkoniecz** [comment]
@phodgson1 - ping as requested

staging link: https://cl-staging.uksouth.cloudapp.azure.com/view/planning/
